src/test/java/com/google/api/codegen/testdata/discoveries/sheets.v4.json.overrides

```diff
@@ -1,12 +1,267 @@
 {
   "java": {
     "methods": {
+      "sheets.spreadsheets.batchUpdate": {
+        "requestBodyType": {
```

Comment:
The risk of deferring is if another API team wants to override and sees this as the reference way to do it, such that either (a) they do it this way and we have more trouble changing it later, or (b) they decide not to use it because it's too complicated. We could probably avoid that with some care in communications. @vchudnov-g your thoughts?